config/install/views.view.stanford_person.yml

```diff
@@ -1,3 +1,4 @@
+uuid: cf1fc1a9-9b4f-4128-b71e-1acc0d673cbd
```

Comment:
No uuids in module files.

Reply:
@sherakama,
How do I best address this issue? Just delete the line, or did I miss something in the development process?

Reply:
Just delete it.